biogtr/models/embedding.py

```diff
@@ -81,9 +159,11 @@ def _sine_box_embedding(
         if self.normalize:
             boxes = boxes / (boxes[:, -1:] + 1e-6) * self.scale
 
-        dim_t = torch.arange(self.features, dtype=torch.float32)
+        dim_t = torch.arange(self.features // 4, dtype=torch.float32)
 
-        dim_t = self.temperature ** (2 * self._torch_int_div(dim_t, 2) / self.features)
+        dim_t = self.temperature ** (
+            2 * self._torch_int_div(dim_t, 2) / self.features // 4
```

Comment:
For clarity on the order of operations:

```suggestion
            (2 * self._torch_int_div(dim_t, 2)) / (self.features // 4)
```